libcore/src/IO/IniFileParser.cpp

```diff
@@ -235,6 +235,18 @@ bool IniFileParser::ParseHeader(TiXmlNode * xHeader)
         xHeader->FirstChildElement("trajectories")->Attribute("fps", &fps);
         _config->SetFps(fps);
 
+
+        unsigned int precision = 0;
+        auto ret               = xHeader->FirstChildElement("trajectories")
+                       ->QueryUnsignedAttribute("precision", &precision);
+
+        if(ret == TIXML_SUCCESS || (ret == TIXML_WRONG_TYPE)) {
+            if(precision < 1 || precision > 6) {
+                LOG_WARNING("Invalid value for precision (should be in [1, 6])", precision);
```

Comment:
Maybe we should also add a note here, that the default precision 2 is used.

Reply:
done